README.md

```diff
@@ -87,11 +90,12 @@ If you want to change it to `and` operation, you can specify:
 Profile.where_languages(:french, :spanish, op: :and) #=> SELECT * FROM profiles WHERE languages = 10
 ```
 
-Negated forms are supported as well, without additional options though.
+The negated form works in a similar way. It searches with `or` operation by default. That means the profiles that have either French or Spanish are excluded from the scope.
+
+If you want to change it to `and` operation to exclude the profiles that have both French and Spanish at the same time, you can specify:
 
 ```ruby
-Profile.where_not_languages(:french)           #=> SELECT * FROM profiles WHERE languages & 8 = 0
-Profile.where_not_languages(:french, :spanish) #=> SELECT * FROM profiles WHERE languages & 10 = 0
+Profile.where_not_languages(:french, :spanish, op: :and) #=> SELECT * FROM profiles WHERE languages & 10 < 10
```

Comment:
Does this really work as intended? Wouldn't `Profile.where_not_languages(:chinese, op: :and)` returns 0, and `0 < 10` becomes always true?